autoscheduler/frontend/src/components/SchedulingPage/ConfigureCard/ConfigureCard.tsx

```diff
@@ -117,6 +129,17 @@ const ConfigureCard: React.FC = () => {
             : 'Generate Schedules'}
         </Button>
       </div>
+      <Snackbar
+        open={showSnackbar}
+        autoHideDuration={10000}
+        message="No schedules found. Try widening your criteria."
+        onClose={(): void => setShowSnackbar(false)}
```

Comment:
Might want to change this callback to the following:
```ts
  const handleSnackbarClose = (_event: any, reason: string): void => {
    if (reason === 'clickaway') return;
    setShowSnackbar(false);
  };
```
Right now the snackbar closes if you click anywhere outside of it, which makes it easy to ignore if you're not paying close attention

Reply:
Hmm, I know this was the recommended behavior in the docs, but I kinda liked having it go away if you clicked outside. Can I get a 3rd opinion from @gannonprudhomme or @ThatJuanGuy ?

Reply:
I pulled the branch but I can't really check how obvious the snack bar is when it pops up because I still generate the same 3 preset schedules regardless of what courses I have selected. Regardless, I don't think it should disappear if you click outside of it. This is because if no schedules are showing up the user might be randomly clicking to pass the time or repeatedly clicking the generate button to try and make it go faster. Both of which would make it much easier for them to miss the message letting them know no schedules can be generated with their current settings.